src/screens/TopicPageScreen/index.js

```diff
@@ -428,10 +395,10 @@ const TopicPageScreen = (props) => {
     <SafeAreaProvider forceInset={{top: 'always'}} style={styles.parentContainer}>
       <NavHeader
         domain={topicName}
-        animatedHeight={animatedHeight}
+        animatedHeight={headerHeight}
         onShareCommunity={onShareCommunity}
         isHeaderHide={isHeaderHide}
-        opacityHeaderAnimation={opacityHeaderAnimation}
+        opacityHeaderAnimation={opacityHeader}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `animatedHeight` and `opacityHeaderAnimation` props have been replaced with `headerHeight` and `opacityHeader`. Ensure that the `NavHeader` component correctly handles these new props.
```diff
-        animatedHeight={animatedHeight}
+        animatedHeight={headerHeight}
         onShareCommunity={onShareCommunity}
         isHeaderHide={isHeaderHide}
-        opacityHeaderAnimation={opacityHeaderAnimation}
+        opacityHeaderAnimation={opacityHeader}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->